{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:07:42"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:07:42"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:07:42"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:07:54"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:07:54"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:07:54"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:14:49"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:14:49"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:14:49"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:14:57"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:14:57"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:14:57"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:15:21"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:15:21"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:15:21"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:16:38"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:16:38"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:16:38"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:16:50"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:16:50"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:16:50"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:16:59"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:16:59"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:16:59"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:17:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:17:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:17:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:17:30"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:17:30"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:17:30"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:18:06"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:18:06"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:18:06"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:18:22"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:18:22"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:18:22"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:19:22"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:19:22"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:19:22"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:19:39"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:19:39"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:19:39"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:19:42"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:19:42"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:19:42"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:19:54"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:19:54"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:19:54"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:20:20"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:20:20"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:20:20"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:20:38"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:20:38"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:20:38"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:21:07"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:21:07"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:21:07"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:22:01"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:22:01"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:22:01"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:22:05"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:22:05"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:22:05"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:23:29"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:23:29"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:23:29"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:23:46"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:23:46"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:23:46"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:25:35"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:25:35"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:25:35"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:26:35"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:26:35"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:26:35"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:27:25"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:27:25"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:27:25"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:27:51"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:27:52"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:27:52"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:27:56"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:27:56"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:27:56"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:27:58"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:27:58"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:27:58"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:29:24"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:29:24"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:29:24"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:29:41"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:29:41"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:29:41"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:30:04"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:30:04"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:30:04"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:30:07"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:30:07"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:30:07"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:30:33"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:30:33"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:30:33"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:30:48"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:30:48"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:30:48"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:33:12"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:33:12"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:33:12"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:33:25"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:33:25"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:33:25"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:33:42"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:33:42"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:33:42"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:34:30"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:34:31"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:34:31"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:34:37"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:34:37"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:34:37"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:35:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:35:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:35:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:35:13"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:35:13"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:35:13"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:37:22"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:37:22"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:37:22"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 21:37:39"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 21:37:40"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 21:37:40"}
